diff --git a/Makefile.am b/Makefile.am
index aa1f59b..0bbbef6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -209,11 +209,7 @@ install-data-hook:
 uninstall-hook:
 	@rm -f  $(DESTDIR)$(themedir)/icon-theme.cache

-pkgconfigdir = $(datadir)/pkgconfig
-pkgconfig_DATA = adwaita-icon-theme.pc
-
 EXTRA_DIST =			\
-	adwaita-icon-theme.pc.in  \
 	Adwaita			\
 	COPYING_CCBYSA3		\
 	COPYING_LGPL		\
diff --git a/configure.ac b/configure.ac
index d855b7a..4f6ab3b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -80,7 +80,6 @@ AM_CONDITIONAL([ENABLE_L_XL_VARIANTS], [test x$enable_l_xl_variants = xyes])

 AC_CONFIG_FILES([
 Makefile
-adwaita-icon-theme.pc
 src/Makefile
 src/fullcolor/Makefile
 src/symbolic/Makefile
