diff --git a/app/main.cpp b/app/main.cpp
index a9539b5..4c8d3cd 100644
--- a/app/main.cpp
+++ b/app/main.cpp
@@ -176,7 +176,7 @@ int main(int argc, char **argv)
     aboutData.processCommandLine(&parser);
 
     // This is needed to prevent Dolphin from freezing when opening an archive.
-    KDBusService dbusService(KDBusService::Multiple);
+    KDBusService dbusService(KDBusService::Multiple | KDBusService::NoExitOnFailure);
 
     // Session restoring.
     if (application.isSessionRestored()) {
