diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7ef92f46..413b21e8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -120,10 +120,10 @@ find_package(Qt5Widgets)
 if(Qt5Widgets_FOUND)
   add_subdirectory(widgets)
 endif()
-find_package(Qt5 COMPONENTS Qml Quick)
-if(Qt5Qml_FOUND AND Qt5Quick_FOUND)
-  add_subdirectory(qml)
-endif()
+#find_package(Qt5 COMPONENTS Qml Quick)
+#if(Qt5Qml_FOUND AND Qt5Quick_FOUND)
+#  add_subdirectory(qml)
+#endif()
 
 if(BUILD_EXAMPLES OR BUILD_PLAYERS)
   add_subdirectory(examples)
