diff --git a/CMakeLists.txt b/CMakeLists.txt
index 92b311bfc..c21f0f61b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -180,7 +180,6 @@ ecm_generate_dbus_service_file(
    EXECUTABLE "${KDE_INSTALL_FULL_BINDIR}/dolphin --daemon"
    SYSTEMD_SERVICE plasma-dolphin.service
    DESTINATION ${KDE_INSTALL_DBUSSERVICEDIR}
-   RENAME org.kde.dolphin.FileManager1.service
 )
 
 ecm_install_configured_files(INPUT plasma-dolphin.service.in DESTINATION ${SYSTEMD_USER_UNIT_INSTALL_DIR})
diff --git a/src/main.cpp b/src/main.cpp
index 779690c1c..7956824f4 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -148,7 +148,7 @@ int main(int argc, char **argv)
         QObject::connect(&app, &QGuiApplication::commitDataRequest, disableSessionManagement);
         QObject::connect(&app, &QGuiApplication::saveStateRequest, disableSessionManagement);
 
-        KDBusService dolphinDBusService;
+        KDBusService dolphinDBusService(KDBusService::NoExitOnFailure);
         DBusInterface interface;
         interface.setAsDaemon();
         return app.exec();

