diff --git a/CMakeLists.txt b/CMakeLists.txt
index 49842bd4c..b1d298e92 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -166,9 +166,9 @@
     COMPONENT Devel
 )
 
-configure_file(org.kde.dolphin.FileManager1.service.in
-               ${CMAKE_CURRENT_BINARY_DIR}/org.kde.dolphin.FileManager1.service)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/org.kde.dolphin.FileManager1.service
+configure_file(org.freedesktop.FileManager1.service.in
+               ${CMAKE_CURRENT_BINARY_DIR}/org.freedesktop.FileManager1.service)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/org.freedesktop.FileManager1.service
         DESTINATION ${KDE_INSTALL_DBUSSERVICEDIR})
 install(FILES dolphin.categories  DESTINATION  ${KDE_INSTALL_LOGGINGCATEGORIESDIR})
 
diff --git a/org.kde.dolphin.FileManager1.service.in b/org.freedesktop.FileManager1.service.in
similarity index 100%
rename from org.kde.dolphin.FileManager1.service.in
rename to org.freedesktop.FileManager1.service.in
diff --git a/src/main.cpp b/src/main.cpp
index ef2905d77..56d1789b0 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -145,7 +145,7 @@ extern "C" Q_DECL_EXPORT int kdemain(int argc, char **argv)
         QObject::connect(&app, &QGuiApplication::commitDataRequest, disableSessionManagement);
         QObject::connect(&app, &QGuiApplication::saveStateRequest, disableSessionManagement);
 
-        KDBusService dolphinDBusService;
+        KDBusService dolphinDBusService(KDBusService::NoExitOnFailure);
         DBusInterface interface;
         interface.setAsDaemon();
         return app.exec();

