commit c21a755f8652a0c9fce9f4bb8833a739bf8abf73
Author: Eike Hein <hein@kde.org>
Date:   Wed Sep 6 20:14:47 2017 +0900

    Lower version (HACK)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b5273f6..2002e82 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.0)
 
 set(KF5_VERSION "5.38.0") # handled by release scripts
-set(KF5_DEP_VERSION "5.38.0") # handled by release scripts
+set(KF5_DEP_VERSION "5.37.0") # handled by release scripts
 
 project(kirigami2 VERSION ${KF5_VERSION})
 
@@ -34,7 +34,7 @@ ELSE(STATIC_LIBRARY)
 
 ################# set KDE specific information #################
 
-find_package(ECM 5.38.0 NO_MODULE)
+find_package(ECM 5.37.0 NO_MODULE)
 set_package_properties(ECM PROPERTIES TYPE REQUIRED DESCRIPTION "Extra CMake Modules." URL "https://projects.kde.org/projects/kdesupport/extra-cmake-modules")
 
 # where to look first for cmake modules, before ${CMAKE_ROOT}/Modules/ is checked
