commit 9f4269d2f3b87b3ea349a8cefc055f53b043c02d
Author: Elvis Angelaccio <elvis.angelaccio@kdemail.net>
Date:   Wed Aug 3 22:20:29 2016 +0200

    Add org.kde prefix to app icon

diff --git a/desktop/org.kde.kronometer.desktop b/desktop/org.kde.kronometer.desktop
index f010402..75f288f 100755
--- a/desktop/org.kde.kronometer.desktop
+++ b/desktop/org.kde.kronometer.desktop
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Type=Application
 Exec=kronometer
-Icon=kronometer
+Icon=org.kde.kronometer
 Terminal=false
 Name=Kronometer
 Name[ast]=Kronometer
diff --git a/icons/128-apps-kronometer.png b/icons/128-apps-org.kde.kronometer.png
similarity index 100%
rename from icons/128-apps-kronometer.png
rename to icons/128-apps-org.kde.kronometer.png
diff --git a/icons/48-apps-kronometer.png b/icons/48-apps-org.kde.kronometer.png
similarity index 100%
rename from icons/48-apps-kronometer.png
rename to icons/48-apps-org.kde.kronometer.png
diff --git a/icons/64-apps-kronometer.png b/icons/64-apps-org.kde.kronometer.png
similarity index 100%
rename from icons/64-apps-kronometer.png
rename to icons/64-apps-org.kde.kronometer.png
diff --git a/icons/CMakeLists.txt b/icons/CMakeLists.txt
index 7b56382..d49ff3d 100644
--- a/icons/CMakeLists.txt
+++ b/icons/CMakeLists.txt
@@ -1,7 +1,7 @@
 ecm_install_icons(
     ICONS
-    48-apps-kronometer.png
-    64-apps-kronometer.png
-    128-apps-kronometer.png
-    sc-apps-kronometer.svgz
+    48-apps-org.kde.kronometer.png
+    64-apps-org.kde.kronometer.png
+    128-apps-org.kde.kronometer.png
+    sc-apps-org.kde.kronometer.svgz
     DESTINATION ${KDE_INSTALL_ICONDIR})
diff --git a/icons/sc-apps-kronometer.svgz b/icons/sc-apps-org.kde.kronometer.svgz
similarity index 100%
rename from icons/sc-apps-kronometer.svgz
rename to icons/sc-apps-org.kde.kronometer.svgz
